src/components/Proposals/ProposalPage/OPProposalPage/ProposalVotesCard/OptimisticProposalVotesCard.tsx

```diff
@@ -50,7 +50,7 @@ const OptimisticProposalVotesCard = ({
           </HStack>
         </button>
         <div>
-          <div className="py-0 px-4 font-semibold mb-2">Proposal votes</div>
+          <div className="py-0 px-4 font-semibold mb-2">Voting Activity</div>
```

Comment:
super small nit but I typically like to only upper the first word unless it's a proper title. So in this case, `Voting activity`